files/config-securedrop-4.4

```diff
@@ -7472,7 +7476,7 @@ CONFIG_TIMER_STATS=y
 # CONFIG_DEBUG_ATOMIC_SLEEP is not set
 # CONFIG_DEBUG_LOCKING_API_SELFTESTS is not set
 # CONFIG_LOCK_TORTURE_TEST is not set
-CONFIG_STACKTRACE=y
+# CONFIG_STACKTRACE is not set
```

Comment:
This setting is likely the one that "fixes" the error in the logs, but if so, would actually suppress it instead of solving. Confirmed with a staging VM that this setting is the default in current 3.14 kernels.